src/content/docs/swate_videos.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
```

Comment:
Most of these are outdated or the feature is currently not actively developed (Adding a checklist to your table). I don't think the current iteration of videos should be promoted.